subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,76 @@
+
+#### Comprehension
+
+##### Describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
+
+##### Ask the student to explain one of the Ansible playbook
+
+###### did he/she clearly explain all the Ansible playbook
+
+##### Discuss the CI/CD pipeline setup.
+
+###### Was the CI/CD pipeline, integrating Jenkins and SonarQube, effectively used to improve the development process?
+
+##### Detail the security measures implemented.
+
+###### Were comprehensive security measures like SSL/TLS, secret management, and principle of least privilege correctly implemented?
+
+##### Explain the database schema for PostgreSQL and Neo4j.
+
+###### Did the data structure in PostgreSQL and Neo4j effectively support the application's requirements?
+
+#### Functional
+
+##### Verify the execution of Ansible playbooks.
+
+###### Did the Ansible playbooks execute without errors and configure the environment as intended?
+
+###### Were the playbooks able to handle re-running scenarios without causing disruptions or inconsistencies?
```

Comment:
How can you trigger a "re-running scenario"? I would add this information to ease the audit process

Reply:
`Is there a definition of how to "recover" if something bad happens in the playbook`

Reply:
This point is still missing

Reply:
by creating an error in the environment they can  can trigger a re-run. i don't really see that i have to add that both of the student will work on the project and the auditor can ask the student to trigger it in case he doesn't know.
there is multiple ways or scenarios to recover.